.github/workflows/main.yml

```diff
@@ -36,6 +55,7 @@ jobs:
         name: results
         path: |
           commandTree.json
+          zowe.json
```

Comment:
For the real contribution, I think the only one we will really care about is the generated webhelp. This is great to prove the PoC works though!